packages/extension-starter/package-lock.json

```diff
@@ -1,8589 +0,0 @@
-{
-  "name": "@salesforce/extension-starter",
```

Comment:
Aside: there are a few packages that were meant to be deleted... but some files inside them were still tracked by git. So I deleted them.